js/Numbers/4-D6.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var rollDice = function(sides) {
```

Comment:
This function has a subtle bug, leading to two bad things happening - when dealing with  random stuff test in a loop if you can, with many results (1000s). Try this test harness and fix your code:
````

var stats = {};
var roll;
for (var i =0; i < 100000; i++) {
    roll = rollDice(6);
    stats[roll] = stats[roll] ? stats[roll] +1 : 1;
}
console.log(stats);
````